### PR TITLE
Created the Portoflio and About me sections

### PR DESCRIPTION
**Pull request by MustakimMasum**
Activities in this branch:

- Portfolio section added
- About me section added
- Used Grid layout for portfolio section
- used flex layout for about me section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -50,5 +50,174 @@ <h2>Let's connect</h2>
...
+            <div>
+                <h1>Tonic</h1>
+                <div class="summary">
+                    <h2 class="canopy">CANOPY</h2>
+                    <div class="counter"></div>
+                    <h2 class="caption">Back End Dev</h2>
+                    <div class="counter"></div>
+                    <h2 class="caption">2015</h2>                
+                </div>
+                <p>
```
Comment (abdulhamiid) [inline index.html]: I suggest that we make the container of the work section a `section` tag.

* It represents a generic standalone section of the document, so let's use it 🤗. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.

File: index.html [hunk 2]
```diff
@@ -50,5 +50,174 @@ <h2>Let's connect</h2>
...
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="techlist">
+                    <li class="tech">HTML</li>
+                    <li class="tech">CSS</li>
+                    <li class="tech">JavaScript</li>
+                </ul>
```
Comment (abdulhamiid) [inline index.html]: Your design
image

Figma
image

According to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md), please adjust your design to ensure that it is similar to the guideline as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). 💪

File: index.html [hunk 3]
```diff
@@ -50,5 +50,174 @@ <h2>Let's connect</h2>
...
+                <button class="project">See project</button>
+            </div>
```
Comment (abdulhamiid) [inline index.html:78]: According to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md), kindly implement the button by following the button styling: `Active`, `Hover` and `Pressed` 🤗💪

(see design) 

image

File: index.html [hunk 4]
```diff
@@ -50,5 +50,174 @@ <h2>Let's connect</h2>
...
         </section>
     </div>
+    <div class="works">
+        <section class="portfolio">
+            <div>
+                <img src="Assets/Tonic1.png" alt="Portfolio image">
+            </div>
+            <div>
+                <h1>Tonic</h1>
+                <div class="summary">
```
Comment (abdulhamiid) [inline index.html]: Your design
image

Figma
image

- I would suggest that we use a header tag `hx` (h2, h3, ...) for the title (e.g., "Tonic"). The `<h1>` to `<h6>` tags are used to define HTML headings. However, `<h1>` defines the most important heading (i.e.  The main title of the page). `<h6>` defines the least important heading.  [Check this link](https://www.reliablesoft.net/h1-tag/) for more info

- Kindly use the same color as the design on Figma for the header so as to stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). 💪

File: index.html [hunk 5]
```diff
@@ -50,5 +50,174 @@ <h2>Let's connect</h2>
...
+                    <div>
+                        <img src="Assets/CSS.png" alt="CSS">
+                    </div>
+                    <div>
+                        <h3>CSS</h3>
+                    </div>
+                </li>
+            </ul>
+            <h6>Frameworks</h6>
+            <h6>Skills</h6>
```
Comment (abdulhamiid) [inline index.html]: According to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md), please use `ul` and `li` for categories in "About myself" section. i.e. `Languages`, `Frameworks`, and `Skills` should be list items.


Example
```HTML
<ul>
  <li>
    <h3>Languages</h3>
    . . . 
    <ul>
       <li>JavaScript</li>
       <li> . . .  </li>
       <li> . . .  </li>
    </ul>
  </li>
  <li>
    <h3>Frameworks</h3>
     . . . 
  </li>
  <li> 
   <h3>Skills</h3>
    . . . 
  </li>
</ul>
```

Your design
image

Figma
image

Kindly adjust your design to ensure that your design is similar to the guideline as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). 💪

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @firepicaso 👋, 
Alt Text
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️Understand how to parse a Figma design to create a UI.
- ✔️ Use Flexbox to place elements on the page.
- ✔️ Use CSS Grid to place elements on the page.
- ✔️ Meaningful commit messages
- ✔️ Use GitHub Flow

## Required Changes ♻️
_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 5 above)
Comment (MustakimMasum) [reply index.html]: I am very close to this template.. also running out of time. @abdulhamiid

File: style.css [hunk 6]
```diff
@@ -70,6 +74,84 @@
...
+    color: #7a869a;
+  }
+
+  .techlist {
+    display: flex;
+    list-style: none;
+    justify-content: space-between;
+    padding: 0;
+    width: 75%;
+  }
```
Comment (jmonto55) [inline style.css]: - [ ] Great work styling your tech list. 💯 However, please kindly ensure to apply the same styles provided in the Figma design as requested by the previous reviewer.🤓 
>Hint - pay special attention to `font-family`, `font-size`, `color`, and `padding`.

Figma design. 👇🏽
f


Your design. 👇🏽
d

File: index.html [hunk 7]
```diff
@@ -49,6 +49,175 @@ <h2>Let's connect</h2>
...
         </section>
-    </div>
+    </section>
+    <section class="works">
+        <section class="portfolio">
+            <div>
+                <img src="Assets/Tonic1.png" alt="Portfolio image">
+            </div>
+            <div>
+                <h1 class="tonic">Tonic</h1>
```
Comment (jmonto55) [inline index.html]: - [ ] Great work building the header for your card. 💯 However, please kindly ensure to change the `<h1>` tag for an `<h2>` tag here as requested by the previous reviewer.🤓 This change will boost the accessibility and readability of your web by specifying only one main header for each page.
- [ ] Also, please kindly ensure to implement the same change for all your cards headers and make sure that your `index.html` file has only one main header `<h1>`. 🔧

File: style.css [hunk 8]
```diff
@@ -44,6 +44,10 @@
     line-height: 52px;
     letter-spacing: 0;
     text-align: left;
+    margin: 0;
+  }
+
+  .main h1 {
     margin: 100px 0 0 0;
   }
```
Comment (jmonto55) [inline style.css]: - [ ] Excellent job applying styles to your headers. 🚀 However, please kindly ensure to apply the correct color for them as requested by the previous reviewer. 🤓
>Hint - #172b4d

ff

File: index.html [hunk 9]
```diff
@@ -49,6 +49,175 @@ <h2>Let's connect</h2>
...
+                    <div>
+                        <img src="Assets/CSS.png" alt="CSS">
+                    </div>
+                    <div>
+                        <h3>CSS</h3>
+                    </div>
+                </li>
+            </ul>
+            <h6>Frameworks</h6>
+            <h6>Skills</h6>
```
Comment (jmonto55) [inline index.html]: - [ ] Great work rendering your skills list. 💯 However, please kindly ensure to follow the hint provided by the previous reviewer and use, yet another `<ul>` tag to wrap the internal lists for each of your Languajes .🤓
>Hint 👇🏽
```
<ul>
  <li>
    <h3>Languages</h3>
    . . . 
    <ul>
       <li>JavaScript</li>
       <li> . . .  </li>
       <li> . . .  </li>
    </ul>
  </li>
  <li>
    <h3>Frameworks</h3>
     . . . 
  </li>
  <li> 
   <h3>Skills</h3>
    . . . 
  </li>
</ul>
```

File: style.css [hunk 10]
```diff
@@ -79,4 +161,108 @@
...
+    margin: 15px 0;
+  }
+
+  .technologies {
+    background-color: #f7f7f9;
+    border-radius: 8px;
+    list-style: none;
+    align-items: left;
+    padding: 0 12px;
+  }
```
Comment (jmonto55) [inline style.css]: - [ ] Good work building your Languages section. ✔️ However, please kindly ensure to change the styles as requested by the previous reviewer. 🤓 Currently, I can still see significant differences between your project and the Figma design. 🔧

Figma design. 👇🏽
f


Your design. 👇🏽
d

**Review by jmonto55 — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES 🟡

Hi @firepicaso 👋🏽, 

Great work on making the changes requested by a previous reviewer 👌
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

> 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by wayungi — COMMENTED** [review]
Hi @firepicaso , 
image

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌ ❌ 

- [ ] You have submitted a project that has failing linter tests. Kindly fix this issue & resubmit

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi @firepicaso,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ Linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**